qa/suites/fs/upgrade/upgrade_without_reducing_max_mds/.qa

```diff
@@ -0,0 +1 @@
+../.qa/
```

Comment:
This should be under `qa/suites/fs/upgrade/mds_upgrade_sequence/`

Reply:
I'll remove it from `upgrade_without_reducing_max_mds` as its already part of `mds_upgrade_sequence`